client/src/ui/base/_themes.scss

```diff
@@ -183,6 +183,15 @@
   --form-invalid-focus-color: #{$mdn-color-light-theme-red-50};
   --form-invalid-focus-effect-color: #{rgba($mdn-color-light-theme-red-50, 0.2)};
 
+  --baseline-supported-bg: #e6f4ea;
+  --baseline-supported-engine: #ceead6;
+  --baseline-supported-mark: url("../../assets/icons/baseline/check.svg");
+  --baseline-unsupported-bg: #f1f3f4;
+  --baseline-unsupported-engine: #e3e6e8;
+  --baseline-unsupported-mark: url("../../assets/icons/baseline/cross.svg");
+  --baseline-browser-check: #1e8e3e;
+  --baseline-browser-cross: #666;
```

Comment:
🟡 Can we add postfixes for all variables to make them more clearer?

```suggestion
  --baseline-supported-bg: #e6f4ea;
  --baseline-supported-engine-bg: #ceead6;
  --baseline-supported-image: url("../../assets/icons/baseline/check.svg");
  --baseline-unsupported-bg: #f1f3f4;
  --baseline-unsupported-engine-bg: #e3e6e8;
  --baseline-unsupported-image: url("../../assets/icons/baseline/cross.svg");
  --baseline-browser-check-bg: #1e8e3e;
  --baseline-browser-cross-bg: #666;
```

And should these last two ones be called `--baseline-supported-browser-color/bg` and `--baseline-unsupported-browser-color/bg` instead?

Reply:
For the last two, no: `--baseline-{unsupported|supported}-*` specifies the different values when baseline shows as supported or unsupported, the last two are values per-browser which are related - but not specific to - baseline supported/unsupported: it's possible to have a green browser check in both baseline supported and baseline unsupported for instance. I'll keep the "scoping" as it is, but update the naming to be more consistent and call these something like `--baseline-browser-{unsupported|supported}-*`